docusaurus/website/src/components/CodeSnippets/CodeSnippets.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from 'react';
+import CodeBlock from '@theme/CodeBlock';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+function CodeSnippets({ paths, groupId, queryString }) {
+  const snippetPaths = paths.split(',').map((path) => path.trim());
+  const [snippets, setSnippets] = useState([]);
+
+  useEffect(() => {
+    async function fetchSnippets() {
+      const fetchedSnippets = await Promise.all(
+        snippetPaths.map(async (path) => {
+          const [_, value, language] = path.split('.');
```

Comment:
I would probably add a block of comments above the component definition that explains how to use this component on the call-site, especially how the name of the file relates to the names of the tabs.